src/Random/README.md

```diff
@@ -0,0 +1,62 @@
+# Dafny.Random
+
+## Overview
+
+The `Dafny.Random` module provides a uniform interface to random values across target languages.
+
+```
+    // Return an arbitrary boolean value
+    predicate nextBool()
+
+    // Return an arbitrary integer value in the range [0, bound)
+    function nextInt(bound: int := 0): (value: int)
```

Comment:
Why not make `bound` a required argument? I can see only by looking at the implementation that the implicit default bound is target language dependent, and that doesn't seem terribly useful.